src/Microsoft.Dnx.Runtime.Sources/Impl/EnvironmentNames.cs

```diff
@@ -16,12 +16,9 @@ internal static class EnvironmentNames
         public const string AppBase = "DNX_APPBASE";
```

Comment:
This cannot be removed because dnu publish sets this environment variable in the scripts it generates. Anyone know why we are doing this and if it should be removed?